requireUp.test.js

```diff
@@ -0,0 +1,59 @@
+import path from "node:path";
```

Comment:
Using ESM syntax here because Vitest is ESM-first.

Reply:
If CJS is preferred, Vitest can be swapped out for Jest. I was able to run the tests with Jest with the following changes:

```js
const path = require("node:path");

const { requireUp } = require("./requireUp");
const { DEFAULT_EXTENSIONS } = require("./constants");

// Path to the 'fixtures' directory
const fixturesDir = path.resolve(__dirname, './fixtures');

describe("requireUp", () => {
// Implementation of tests is unchanged
```

Reply:
ESM is great, thanks!